sdks/builds/ios.mk

```diff
@@ -301,29 +301,6 @@ $(eval $(call iOSSimulatorTemplate,sim64,x86_64))
 $(eval $(call iOSSimulatorTemplate,simtv,x86_64))
 $(eval $(call iOSSimulatorTemplate,simwatch,i386))
 
-ifndef IGNORE_PACKAGE_LLVM
-
-# Download a prebuilt llvm
-.stamp-ios-llvm-$(LLVM_HASH):
-	./download-llvm.sh $(LLVM_HASH) $(LLVM_JENKINS_LANE)
```

Comment:
This stamp file contained the version, so changing the version would make it download the new version.